mythril/laser/ethereum/instructions.py

```diff
@@ -1578,18 +1626,92 @@ def log_(self, global_state: GlobalState) -> List[GlobalState]:
         # Not supported
         return [global_state]
 
+    @staticmethod
+    def _create_transaction_helper(
+        global_state, call_value, mem_offset, mem_size, op_code, create2_salt=None
+    ):
+
+        mstate = global_state.mstate
+        environment = global_state.environment
+        world_state = global_state.world_state
+
+        try:
+            call_data = get_call_data(
+                global_state,
+                util.get_concrete_int(mem_offset),
```

Comment:
We should directly use the mem_offset and their size instead of using their concrete values. As this function handles symbolic offsets and sizes